apps/shared/avifjpeg.c

```diff
@@ -568,18 +1149,50 @@ avifBool avifJPEGRead(const char * inputFilename,
         }
         avifImageFixXMP(avif); // Remove one trailing null character if any.
     }
+
+#if defined(AVIF_ENABLE_EXPERIMENTAL_JPEG_GAIN_MAP_CONVERSION)
+    // The primary XMP block (for the main image) must contain a node with an hdrgm:Version field if and only if a gain map is present.
```

Comment:
hdrm*?

Reply:
hdrgm is actually the one that's correct, so I fixed the instances of "hdrm".
Note that I don't actually check that the prefix is "hdrgm", just that there is a prefix associated with the namespace "http://ns.adobe.com/hdr-gain-map/1.0/". This seems more in the spirit of XML to me, where the prefix should be arbitrary. But Adobe's spec says that the prefix should be "hdrgm", not sure why, maybe for simplicity. I don't think it matters much and I expect that it will always be "hdrgm" in practice.